### PR TITLE
Add new Q & A

### PR DESCRIPTION
**Pull request by qw245roar**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 # Frequently Asked Questions
 
-- **What is the difference between score and performance of my Bot?**
+**Q: Why do you choose log-likelihood as the metric to maximize?**
```
Comment (lilyS123) [inline FAQ.md]: to maximize?

File: FAQ.md [hunk 2]
```diff
@@ -2,7 +2,18 @@
...
 
-The score is computed when the true value is revealed and compared to your prediction: a low score is better, exactly like golf! Performance is a weighted average of your scores so we look at an aggregate of your scores to reflect your overall Bot quality.
+A: The log-likelihood is both intuitive and flexible, and an efficient means of statistical inference. Considering that it includes both the mean and variance, it is always used in finance to evaluate how accurate your prediction is from the ground-truth we are interested in. More details can be found here https://en.wikipedia.org/wiki/Maximum_likelihood_estimation.
```
Comment (lilyS123) [inline FAQ.md]: 'always' used in finance or 'commonly' or 'often'?  Want to avoid blanket statements unless 100 true.

File: FAQ.md [hunk 3]
```diff
@@ -2,7 +2,18 @@
...
+
+A: It is the standard deviation of your prediction error, also referred to as sigma in a statistical context.
+
+**Q: Is the precision important for obtaining a good score?**
+
+A: Yes.
+
+**Q: What is the difference between score and performance of my Bot?**
+
+A: The score is computed when the true value is revealed and compared to your prediction: a low score is better, exactly like golf! Performance is a weighted average of your scores so we look at an aggregate of your scores to reflect your overall Bot quality.
```
Comment (lilyS123) [inline FAQ.md]: Want to link them to detailed write up Ilya is preparing explaining the perf calculation?

File: FAQ.md (hunk 1 above)
Comment (qw245roar) [reply FAQ.md]: Yes. maybe add this after 'to maximize' in a bracket ():
(or minimize the minus log-likelihood)

**Comment by AntoineToussaint** [comment]
Is it good to merge?

File: FAQ.md (hunk 2 above)
Comment (qw245roar) [reply FAQ.md]: Yes. 'commonly' sounds more proper.
Yes.  'how accurate your prediction is relative to the ground truth' is good.

**Comment by AntoineToussaint** [comment]
@qw245roar Please make the changes!

**Pull request merged**